### PR TITLE
[DOC] Add missing default values to the docstrings in 'nilearn/plotting'

### PR DESCRIPTION
**Pull request by anupriyakkumari**
Completes the following tasks from https://github.com/nilearn/nilearn/issues/3865:
 Added missing default values in the docstrings of the following files: 

## [nilearn/plotting/matrix_plotting.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/matrix_plotting.py)
[nilearn/plotting/matrix_plotting.py::plot_matrix - line: 217](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/matrix_plotting.py#L217)
[nilearn/plotting/matrix_plotting.py::plot_contrast_matrix - line: 339](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/matrix_plotting.py#L339)
[nilearn/plotting/matrix_plotting.py::plot_design_matrix - line: 459](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/matrix_plotting.py#L459)

## [nilearn/plotting/html_connectome.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_connectome.py)
[nilearn/plotting/html_connectome.py::view_connectome - line: 249](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_connectome.py#L249)
[nilearn/plotting/html_connectome.py::view_markers - line: 355](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_connectome.py#L355)

## [nilearn/plotting/surf_plotting.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/surf_plotting.py)
[nilearn/plotting/surf_plotting.py::plot_surf - line: 730](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/surf_plotting.py#L730)
[nilearn/plotting/surf_plotting.py::plot_surf_contours - line: 1038](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/surf_plotting.py#L1038)
[nilearn/plotting/surf_plotting.py::plot_surf_stat_map - line: 1203](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/surf_plotting.py#L1203)
[nilearn/plotting/surf_plotting.py::plot_img_on_surf - line: 1549](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/surf_plotting.py#L1549)
[nilearn/plotting/surf_plotting.py::plot_surf_roi - line: 1768](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/surf_plotting.py#L1768)

## [nilearn/plotting/html_stat_map.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_stat_map.py)
[nilearn/plotting/html_stat_map.py::view_img - line: 484](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_stat_map.py#L484)

## [nilearn/plotting/img_plotting.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py)
[nilearn/plotting/img_plotting.py::plot_img - line: 331](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L331)
[nilearn/plotting/img_plotting.py::plot_anat - line: 576](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L576)
[nilearn/plotting/img_plotting.py::plot_epi - line: 683](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L683)
[nilearn/plotting/img_plotting.py::plot_roi - line: 823](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L823)
[nilearn/plotting/img_plotting.py::plot_stat_map - line: 1208](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L1208)
[nilearn/plotting/img_plotting.py::plot_glass_brain - line: 1347](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L1347)
[nilearn/plotting/img_plotting.py::plot_connectome - line: 1508](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L1508)
[nilearn/plotting/img_plotting.py::plot_markers - line: 1647](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L1647)
[nilearn/plotting/img_plotting.py::plot_carpet - line: 1813](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L1813)
[nilearn/plotting/img_plotting.py::plot_img_comparison - line: 2077](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/img_plotting.py#L2077)

## [nilearn/plotting/find_cuts.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/find_cuts.py)
[nilearn/plotting/find_cuts.py::find_xyz_cut_coords - line: 31](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/find_cuts.py#L31)

## [nilearn/plotting/html_surface.py](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_surface.py)
[nilearn/plotting/html_surface.py::view_img_on_surf - line: 303](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_surface.py#L303)
[nilearn/plotting/html_surface.py::view_surf - line: 435](https://github.com/nilearn/nilearn/blob/main/nilearn/plotting/html_surface.py#L435)

The functions not listed here did not have parameter description in their docstrings.

### CODE REVIEW

File: doc/changes/latest.rst [hunk 1]
```diff
@@ -101,3 +101,5 @@ Changes
 - :bdg-dark:`Code` Reorder condition in internal call of :func:`nilearn.image.resample_img` to skip checking of array values if interpolation is ``nearest`` (:gh:`4571` by `Jason Kai`_).
 
 - :bdg-dark:`Code` Remove redundant sorting of ``np.unique(data)`` in internal call of :func:`nilearn.image.resample_img` when checking array values (:gh:`4571` by `Jason Kai`_).
+
+- :bdg-primary:`Doc` Add missing default values to the docstrings in 'nilearn/plotting'(:gh:`4660` by `Anupriya Kumari`_).
```
Comment (Remi-Gau) [inline doc/changes/latest.rst:105]: same: we will recombine those entries across PR into a single one

File: nilearn/plotting/find_cuts.py [hunk 2]
```diff
@@ -36,10 +36,10 @@ def find_xyz_cut_coords(img, mask_img=None, activation_threshold=None):
     img : 3D Nifti1Image
         The brain map.
 
-    mask_img : 3D Nifti1Image, optional
+    mask_img : 3D Nifti1Image, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/find_cuts.py:39]:
```suggestion
    mask_img : 3D Nifti1Image or None, default=None
```

File: nilearn/plotting/find_cuts.py [hunk 3]
```diff
@@ -36,10 +36,10 @@ def find_xyz_cut_coords(img, mask_img=None, activation_threshold=None):
...
         An optional brain mask, provided mask_img should not be empty.
 
-    activation_threshold : float, optional
+    activation_threshold : float, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/find_cuts.py:42]:
```suggestion
    activation_threshold : :obj:`float` or None, default=None
```

File: nilearn/plotting/html_connectome.py [hunk 4]
```diff
@@ -274,7 +274,7 @@ def view_connectome(
     node_color : color or sequence of colors, default='auto'
         Color(s) of the nodes.
 
-    edge_threshold : str, number or None, optional
+    edge_threshold : str, number or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_connectome.py:277]:
```suggestion
    edge_threshold : :obj:`str`, number or None, default=None
```

File: nilearn/plotting/html_connectome.py [hunk 5]
```diff
@@ -303,7 +303,7 @@ def view_connectome(
     colorbar_fontsize : int, default=25
         Fontsize of the colorbar tick labels.
 
-    title : str, optional
+    title : str, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_connectome.py:306]:
```suggestion
    title : :obj:`str` or None, default=None
```

File: nilearn/plotting/html_connectome.py [hunk 6]
```diff
@@ -367,18 +367,18 @@ def view_markers(
     marker_coords : ndarray, shape=(n_nodes, 3)
         The coordinates of the nodes in :term:`MNI` space.
 
-    marker_color : ndarray, shape=(n_nodes,), optional
+    marker_color : ndarray, shape=(n_nodes,), default='auto'
```
Comment (Remi-Gau) [inline nilearn/plotting/html_connectome.py:370]:
```suggestion
    marker_color : ndarray of shape=(n_nodes,) or 'auto', default='auto'
```

File: nilearn/plotting/html_connectome.py [hunk 7]
```diff
@@ -367,18 +367,18 @@ def view_markers(
...
         colors of the markers: list of strings, hex rgb or rgba strings, rgb
         triplets, or rgba triplets (i.e. formats accepted by matplotlib, see
         https://matplotlib.org/users/colors.html#specifying-colors)
 
     marker_size : float or array-like, default=5.0
         Size of the markers showing the seeds in pixels.
 
-    marker_labels : list of str, shape=(n_nodes), optional
+    marker_labels : list of str, shape=(n_nodes), default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_connectome.py:378]:
```suggestion
    marker_labels : :obj:`list` of :obj:`str` of shape=(n_nodes)\
                     or None, default=None
```

File: nilearn/plotting/html_connectome.py [hunk 8]
```diff
@@ -367,18 +367,18 @@ def view_markers(
...
         Labels for the markers: list of strings
 
-    title : str, optional
+    title : str, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_connectome.py:381]:
```suggestion
    title : :obj:`str` or None, default=None
```

File: nilearn/plotting/html_stat_map.py [hunk 9]
```diff
@@ -513,15 +513,15 @@ def view_img(
         To turn off background image, just pass "bg_img=False".
         Default='MNI152'.
 
-    cut_coords : None, or a tuple of floats
+    cut_coords : None, or a tuple of floats, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_stat_map.py:516]:
```suggestion
    cut_coords : None, or a :obj:`tuple` of :obj:`float`, default=None
```

File: nilearn/plotting/html_stat_map.py [hunk 10]
```diff
@@ -513,15 +513,15 @@ def view_img(
...
         The :term:`MNI` coordinates of the point where the cut is performed
         as a 3-tuple: (x, y, z). If None is given, the cuts are calculated
         automatically.
 
     colorbar : boolean, default=True
         If True, display a colorbar on top of the plots.
     %(title)s
-    threshold : string, number or None, default=1e-6
+    threshold : string, number or None, default=1e-06
```
Comment (Remi-Gau) [inline nilearn/plotting/html_stat_map.py:524]:
```suggestion
    threshold : :obj:`str`, number or None, default=1e-06
```

File: nilearn/plotting/html_stat_map.py [hunk 11]
```diff
@@ -547,14 +547,14 @@ def view_img(
         or an anatomical image.
     %(dim)s
         Default='auto'.
-    vmax : float, or None, optional
+    vmax : float, or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_stat_map.py:550]:
```suggestion
    vmax : :obj:`float`, or None, default=None
```

File: nilearn/plotting/html_stat_map.py [hunk 12]
```diff
@@ -547,14 +547,14 @@ def view_img(
...
         max value for mapping colors.
         If vmax is None and symmetric_cmap is True, vmax is the max
         absolute value of the volume.
         If vmax is None and symmetric_cmap is False, vmax is the max
         value of the volume.
 
-    vmin : float, or None, optional
+    vmin : float, or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_stat_map.py:557]:
```suggestion
    vmin : :obj:`float`, or None, default=None
```

File: nilearn/plotting/html_stat_map.py [hunk 13]
```diff
@@ -563,7 +563,7 @@ def view_img(
     %(resampling_interpolation)s
         Default='continuous'.
 
-    width_view : int, optional
+    width_view : int, default=600
```
Comment (Remi-Gau) [inline nilearn/plotting/html_stat_map.py:566]:
```suggestion
    width_view : :obj:`int`, default=600
```

File: nilearn/plotting/html_surface.py [hunk 14]
```diff
@@ -334,7 +334,7 @@ def view_img_on_surf(
         containing inflated and pial meshes, and sulcal depth values for left
         and right hemispheres.
 
-    threshold : str, number or None, optional
+    threshold : str, number or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:337]:
```suggestion
    threshold : :obj:`str`, number or None, default=None
```

File: nilearn/plotting/html_surface.py [hunk 15]
```diff
@@ -354,11 +354,11 @@ def view_img_on_surf(
     %(darkness)s
         Default=1.
 
-    vmax : float or None, optional
+    vmax : float or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:357]:
```suggestion
    vmax : :obj:`float` or None, default=None
```

File: nilearn/plotting/html_surface.py [hunk 16]
```diff
@@ -354,11 +354,11 @@ def view_img_on_surf(
...
         upper bound for the colorbar. if None, use the absolute max of the
         brain map.
 
-    vmin : float or None, optional
+    vmin : float or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:361]:
```suggestion
    vmin : :obj:`float` or None, default=None
```

File: nilearn/plotting/html_surface.py [hunk 17]
```diff
@@ -378,7 +378,7 @@ def view_img_on_surf(
     colorbar_fontsize : int, default=25
         Fontsize of the colorbar tick labels.
 
-    title : str, optional
+    title : str, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:381]:
```suggestion
    title : :obj:`str`, default=None
```

File: nilearn/plotting/html_surface.py [hunk 18]
```diff
@@ -462,7 +462,7 @@ def view_surf(
         of the :term:`mesh` vertices, the second containing the indices
         (into coords) of the :term:`mesh` :term:`faces`.
 
-    surf_map : str or numpy.ndarray, optional
+    surf_map : str or numpy.ndarray, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:465]:
```suggestion
    surf_map : :obj:`str` or :obj:`numpy.ndarray`, default=None
```

File: nilearn/plotting/html_surface.py [hunk 19]
```diff
@@ -482,7 +482,7 @@ def view_surf(
     %(darkness)s
         Default=1.
 
-    threshold : str, number or None, optional
+    threshold : str, number or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:485]:
```suggestion
    threshold : :obj:`str`, number or None, default=None
```

File: nilearn/plotting/html_surface.py [hunk 20]
```diff
@@ -502,11 +502,11 @@ def view_surf(
         Make colormap symmetric (ranging from -vmax to vmax).
         Set it to False if you are plotting a surface atlas.
 
-    vmax : float or None, optional
+    vmax : float or None, default=None
         upper bound for the colorbar. if None, use the absolute max of the
         brain map.
 
-    vmin : float or None, optional
+    vmin : float or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:509]:
```suggestion
    vmin : :obj:`float` or None, default=None
```

File: nilearn/plotting/html_surface.py [hunk 21]
```diff
@@ -522,7 +522,7 @@ def view_surf(
     colorbar_fontsize : int, default=25
         Fontsize of the colorbar tick labels.
 
-    title : str, optional
+    title : str, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/html_surface.py:525]:
```suggestion
    title : :obj:`str`, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 22]
```diff
@@ -705,7 +705,7 @@ def plot_epi(
 
     Parameters
     ----------
-    epi_img : a nifti-image like object or a filename, optional
+    epi_img : a nifti-image like object or a filename, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:708]:
```suggestion
    epi_img : a Niimg-like object or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 23]
```diff
@@ -881,7 +881,7 @@ def plot_roi(
         Default='auto'.
     colorbar : boolean, default=False
         If True, display a colorbar on the right of the plots.
-    cbar_tick_format: str, default="%%.2g" (scientific notation)
+    cbar_tick_format: str, default="%%i"
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:884]:
```suggestion
    cbar_tick_format: :obj:`str`, default="%%i"
```

File: nilearn/plotting/img_plotting.py [hunk 24]
```diff
@@ -1555,14 +1555,14 @@ def plot_connectome(
     edge_cmap : colormap, default=cm.bwr
         Colormap used for representing the strength of the edges.
 
-    edge_vmin, edge_vmax : float, optional
+    edge_vmin, edge_vmax : float, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1558]:
```suggestion
    edge_vmin, edge_vmax : :obj:`float` or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 25]
```diff
@@ -1555,14 +1555,14 @@ def plot_connectome(
...
         If not None, either or both of these values will be used to
         as the minimum and maximum values to color edges. If None are
         supplied the maximum absolute value within the given threshold
         will be used as minimum (multiplied by -1) and maximum
         coloring levels.
 
-    edge_threshold : str or number, optional
+    edge_threshold : str or number, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1565]:
```suggestion
    edge_threshold : :obj:`str`, number or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 26]
```diff
@@ -1585,10 +1585,10 @@ def plot_connectome(
     alpha : float between 0 and 1, default=0.7
         Alpha transparency for the brain schematics.
 
-    edge_kwargs : dict, optional
+    edge_kwargs : dict, default=None
         Will be passed as kwargs for each edge matlotlib Line2D.
 
-    node_kwargs : dict, optional
+    node_kwargs : dict, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1591]:
```suggestion
    node_kwargs : :obj:`dict` or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 27]
```diff
@@ -1585,10 +1585,10 @@ def plot_connectome(
     alpha : float between 0 and 1, default=0.7
         Alpha transparency for the brain schematics.
 
-    edge_kwargs : dict, optional
+    edge_kwargs : dict, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1588]:
```suggestion
    edge_kwargs : :obj:`dict` or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 28]
```diff
@@ -1678,22 +1678,22 @@ def plot_markers(
...
-    node_size : 'auto' or scalar or array-like, optional
+    node_size : 'auto' or scalar or array-like, default='auto'
         Size(s) of the nodes in points^2. By default the size of the node is
         inversely proportional to the number of nodes.
 
     node_cmap : str or colormap, default=plt.cm.viridis_r.
         Colormap used to represent the node measure.
 
-    node_vmin : float, optional
+    node_vmin : float, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1688]:
```suggestion
    node_vmin : :obj:`float`, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 29]
```diff
@@ -1714,7 +1714,7 @@ def plot_markers(
     %(annotate)s
     %(black_bg)s
         Default=False.
-    node_kwargs : dict, optional
+    node_kwargs : dict, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1717]:
```suggestion
    node_kwargs : :obj:`dict` or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 30]
```diff
@@ -1843,7 +1843,7 @@ def plot_carpet(
         If not specified, a new mask will be derived from data.
         See :ref:`extracting_data`.
 
-    mask_labels : :obj:`dict`, optional
+    mask_labels : :obj:`dict`, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:1846]:
```suggestion
    mask_labels : :obj:`dict` or None, default=None
```

File: nilearn/plotting/img_plotting.py [hunk 31]
```diff
@@ -2114,10 +2114,10 @@ def plot_img_comparison(
     src_label : str, default='image set 2'
         Name of source images.
 
-    output_dir : string, optional
+    output_dir : string, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/img_plotting.py:2117]:
```suggestion
    output_dir : :obj:`str` or None, default=None
```

File: nilearn/plotting/matrix_plotting.py [hunk 32]
```diff
@@ -469,7 +469,7 @@ def plot_design_matrix(
     rescale : :obj:`bool`, default=True
         Rescale columns magnitude for visualization or not.
 
-    axes : :class:`matplotlib.axes.Axes`, optional
+    axes : :class:`matplotlib.axes.Axes`, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/matrix_plotting.py:472]:
```suggestion
    axes : :class:`matplotlib.axes.Axes` or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 33]
```diff
@@ -769,15 +769,15 @@ def plot_surf(
         of the :term:`mesh` :term:`faces`,
         or a Mesh object with "coordinates" and "faces" attributes.
 
-    surf_map : str or numpy.ndarray, optional
+    surf_map : str or numpy.ndarray, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:772]:
```suggestion
    surf_map : :obj:`str` or :class:`numpy.ndarray` or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 34]
```diff
@@ -769,15 +769,15 @@ def plot_surf(
...
         Data to be displayed on the surface :term:`mesh`.
         Can be a file
         (valid formats are .gii, .mgz, .nii, .nii.gz,
         or Freesurfer specific files such as
         .thickness, .area, .curv, .sulc, .annot, .label) or
         a Numpy array with a value for each :term:`vertex` of the `surf_mesh`.
 
-    bg_map : str or numpy.ndarray, optional
+    bg_map : str or numpy.ndarray, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:780]:
```suggestion
    bg_map : :obj:`str` or :class:`numpy.ndarray` or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 35]
```diff
@@ -834,7 +834,7 @@ def plot_surf(
         If a number is given, it is used to threshold the image, values
         below the threshold (in absolute value) are plotted as transparent.
 
-    alpha : float or 'auto', default='auto'
+    alpha : float or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:837]:
```suggestion
    alpha : :obj:`float` or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 36]
```diff
@@ -852,7 +852,7 @@ def plot_surf(
 
     %(vmin)s
     %(vmax)s
-    cbar_vmin, cbar_vmax : float, float, optional
+    cbar_vmin, cbar_vmax : float, float, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:855]:
```suggestion
    cbar_vmin, cbar_vmax : float, float, default=None
```

Do you think you could split this in 2?


```
cbar_vmin : :obj:`float` or None, default=None
        Lower bound for the colorbar.
        If None, the value will be set from the data.

cbar_vmin : :obj:`float` or None, default=None
        Lower bound for the colorbar.
        If None, the value will be set from the data.
```

File: nilearn/plotting/surf_plotting.py [hunk 37]
```diff
@@ -876,7 +876,7 @@ def plot_surf(
         .. versionadded:: 0.9.0
 
     %(output_file)s
-    axes : instance of matplotlib axes, None, optional
+    axes : instance of matplotlib axes, None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:879]:
```suggestion
    axes : instance of matplotlib axes or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 38]
```diff
@@ -1074,23 +1074,24 @@ def plot_surf_contours(
         and zero inside ROI,
         or an integer giving the label number for atlases.
 
-    axes : instance of matplotlib axes, None, optional
+    axes : instance of matplotlib axes, None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1077]:
```suggestion
    axes : instance of matplotlib axes or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 39]
```diff
@@ -1074,23 +1074,24 @@ def plot_surf_contours(
...
         The axes instance to plot to. The projection must be '3d' (e.g.,
         `figure, axes = plt.subplots(subplot_kw={'projection': '3d'})`,
         where axes should be passed.).
         If None, uses axes from figure if available, else creates new axes.
     %(figure)s
-    levels : list of integers, or None, optional
+    levels : list of integers, or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1083]:
```suggestion
    levels : list of integers, or None, default=None
```
```suggestion
    levels : :obj:`list` of :obj:`int`, or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 40]
```diff
@@ -1074,23 +1074,24 @@ def plot_surf_contours(
...
         A list of indices of the regions that are to be outlined.
         Every index needs to correspond to one index in roi_map.
         If None, all regions in roi_map are used.
 
-    labels : list of strings or None, or None, optional
+    labels : list of strings or None, or None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1088]:
```suggestion
    labels : :obj:`list` of :obj:`str` or None, or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 41]
```diff
@@ -1074,23 +1074,24 @@ def plot_surf_contours(
...
         A list of labels for the individual regions of interest.
         Provide None as list entry to skip showing the label of that region.
         If None no labels are used.
 
-    colors : list of matplotlib color names or RGBA values, or None, optional
+    colors : list of matplotlib color names or RGBA values, or None \
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1093]:
```suggestion
    colors : :obj:`list` of matplotlib color names or RGBA values, or None \
```

File: nilearn/plotting/surf_plotting.py [hunk 42]
```diff
@@ -1249,7 +1250,7 @@ def plot_surf_stat_map(
         .thickness, .area, .curv, .sulc, .annot, .label) or
         a Numpy array with a value for each :term:`vertex` of the `surf_mesh`.
 
-    bg_map : str or numpy.ndarray, optional
+    bg_map : str or numpy.ndarray, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1253]:
```suggestion
    bg_map : :obj:`str` or :class:`numpy.ndarray` or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 43]
```diff
@@ -1329,7 +1330,7 @@ def plot_surf_stat_map(
         .. versionadded:: 0.9.0
 
     %(output_file)s
-    axes : instance of matplotlib axes, None, optional
+    axes : instance of matplotlib axes, None, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1333]:
```suggestion
    axes : instance of matplotlib axes or None, default=None
```

File: nilearn/plotting/surf_plotting.py [hunk 44]
```diff
@@ -1814,7 +1815,7 @@ def plot_surf_roi(
         The value at each vertex one inside the ROI and zero inside ROI, or an
         integer giving the label number for atlases.
 
-    bg_map : str or numpy.ndarray, optional
+    bg_map : str or numpy.ndarray, default=None
```
Comment (Remi-Gau) [inline nilearn/plotting/surf_plotting.py:1818]:
```suggestion
    bg_map :  :obj:`str` or :class:`numpy.ndarray` or None, default=None
```

**Review by bthirion — COMMENTED** [review]
LGTM so far. Is it ready for merge ?

**Comment by Remi-Gau** [comment]
@anupriyakkumari 
can you fix the styling issues?

should just be matter of running this and committing the changes

```bash
pip install tox
tox
```

**Comment by anupriyakkumari** [comment]
@Remi-Gau I'd have to edit the changelog too. For now I have fixed the styling issue.

**Review by bthirion — APPROVED** [review]
LGTM, but could not check the doc build... to be checked when it completes.

**Comment by anupriyakkumari** [comment]
Hi @Remi-Gau and @bthirion, All checks have passed now. I have also updated the changelog. Let me know if I should do anything else.

**Comment by bthirion** [comment]
Good with me, thx !

**Pull request merged**